Sources/Bridges/OTelSwiftLog/LogHandler.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
+import OpenTelemetryApi
+import OpenTelemetrySdk
+import Logging
+
+// let the bridgename be the url of the package?
+let bridgename: String = "OTelSwiftLog"
+let version: String = "1.0.0"
+
+public struct InstrumentationScope{
```

Comment:
InstrumentationScope is a pre-existing construct in OpenTelemetry and adding this struct is confusing. The loggerBuilder provides all these options, which should be used rather than extending the builder.